content/chat/rooms/index.textile

```diff
@@ -91,10 +91,18 @@ blang[javascript,swift,kotlin].
   ```
 
   ```[kotlin]
-  val presence = PresenceOptions(enter = false)
-  val typing = TypingOptions(timeoutMs = 5_000) // 5 seconds
-  val options = RoomOptions(presence = presence, typing = typing)
-  val room = chatClient.rooms.get(roomId = "basketball-stream", options = options)
+  val room = chatClient.rooms.get(roomId = "basketball-stream") {
+    typing {
+      heartbeatThrottle = 5.seconds
+    }
+    presence {
+      enableEvents = true
+    }
+    reactions()
```

Comment:
I think it's better to put some options or remove `reactions`, with new logic, I don't think we should even allow calling `reactions()` without arguments